### ISSUE TITLE
Where do SafeLinks and SafeAttachments go in the table of applying policies?

### ISSUE THREAD

**Issue by chrislehr** [MicrosoftDocs/microsoft-365-docs#11456]
Subject describes the need - Safe attachments/Safelinks processing - where in this order do they fall?  Article is absent of information on those policies.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d4194fdd-c5de-d424-319a-5e83774b36c2
* Version Independent ID: 83997ad4-a9bd-146e-5da0-8022458cf3ea
* Content: [Order and precedence of email protection - Office 365](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/how-policies-and-protections-are-combined?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/how-policies-and-protections-are-combined.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/how-policies-and-protections-are-combined.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@chrislehr, this issue is fundamentally the same as #8514, but I'll leave it open for now.

**Comment by chrislehr** [comment]
I disagree its the same.  HC Phishing is 100% within the "anti spam policies" SafeLinks and SafeAttachments are their own policies.

**Comment by chrisda** [comment]
@chrislehr, the verdicts or the seriousness of the verdicts aren't what I was talking about. The similarity is that both of these issues are saying a feature/verdict is missing from the order of precedence information.

**Comment by chrislehr** [comment]
That's fair.  I agree they are similar to that extent.  😄
________________________________

**Comment by officedocspr5** [comment]
To make it easier for you to submit feedback on articles on learn.microsoft.com, we're transitioning our feedback system from GitHub Issues to a new experience. 

As part of the transition, this GitHub Issue will be moved to a private repository. We're moving Issues to another repository so we can continue working on Issues that were open at the time of the transition. When this Issue is moved, you'll no longer be able to access it.

If you want to provide additional information before this Issue is moved, please update this Issue before December 15th, 2023.

With the new experience, you no longer need to sign in to GitHub to enter and submit your feedback. Instead, you can choose directly on each article's page whether the article was helpful. Then you can then choose one or more reasons for your feedback and optionally provide additional context before you select **Submit**.

Here's what the new experience looks like.

**Note**: The new experience is being rolled out across learn.microsoft.com in phases. If you don't see the new experience on an article, please check back later.

First, select whether the article was helpful:

![Image showing a dialog asking if the article was helpful with yes and no answers.](https://learn.microsoft.com/Office/media/internal/new-feedback-yes-no.jpg)

Then, choose at least one reason for your feedback and optionally provide additional details about your feedback:

| Article was helpful | Article was unhelpful |
|:--|:---|
|![Image showing a dialog asking how the article was helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-helpful.jpg) | ![Image showing a dialog asking how the article wasn't helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-unhelpful.jpg)|

Finally, select **Submit** and you're done!
